pkg/cmd/server/origin/master.go

```diff
@@ -875,6 +893,23 @@ func (c *MasterConfig) getRequestContextMapper() kapi.RequestContextMapper {
 	return c.RequestContextMapper
 }
 
+// getRequestInfoResolver returns a request resolver.
+func (c *MasterConfig) getRequestInfoResolver() *apiserver.RequestInfoResolver {
```

Comment:
Let me know when you start the master.go rebase. We have moved around so much stuff, hopefully to the better.